packages-exp/analytics-exp/package.json

```diff
@@ -63,5 +62,14 @@
       ".ts"
     ],
     "reportDir": "./coverage/node"
-  }
-}
+  },
+  "exports": {
+    "main": "./dist/index.cjs.js",
+    "esm5": "./dist/index.esm.js",
+    "browser": "./dist/index.esm2017.js",
+    "module": "./dist/index.esm2017.js",
+    "node": "./dist/index.cjs.js",
+    "default": "./dist/index.esm2017.js"
```

Comment:
Should we assign a browser build to default? IIUC, `default` is used by Nodejs if nothing else matches, so should it be a node build? or should we just leave it since there is a `node` field?

Reply:
The node examples always put a browser bundle in the default field. I think they assume any Node app will pick up what's in the Node field.